instagram-app/src/components/CommentSection.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './CommentSection.css';
+
+class CommentSection extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        input: ""
+      };
+    }
```

Comment:
It would be better to place this input variable in your application state so that you can use it in other components.